tsconfig.json

```diff
@@ -4,7 +4,8 @@
 		"target": "ES2020",
 		"outDir": "out",
 		"lib": [
-			"ES2020"
+			"ES2020",
+			"DOM"
```

Comment:
As I already explained: It's because adding axios forces us to do this. Upgrading the node version (as the logs suggest), don't really allow husky to commit

Reply:
Axios was added before july 11. It's already [in this commit](https://github.com/watermelontools/wm-extension/blob/1a78348b12eba8bf1627763c3563cdc3bcae3ef2/package.json) from that date.

Reply:
> As I already explained: It's because adding axios forces us to do this. Upgrading the node version (as the logs suggest), don't really allow husky to commit

So this is not needed in earlier versions of node? When was the requirement introduced?